Gemfile

```diff
@@ -8,13 +8,15 @@ DECIDIM_VERSION = { git: "https://github.com/CodiTramuntana/decidim.git", branch
 
 gem "decidim-conferences", DECIDIM_VERSION
 gem "decidim-consultations", DECIDIM_VERSION
+gem "decidim-decidim_awesome", "~> 0.8.0"
```

Comment:
why not install 0.8.2? https://github.com/Platoniq/decidim-module-decidim_awesome/blob/main/CHANGELOG.md